README.md

```diff
@@ -1,22 +1,20 @@
 # Starter Code: Robust server structure
 
-This server is intended to be run for some checkpoints in the Thinkful curriculum. If you have trouble getting the server to run, reach out to your mentor.
+This server is intended to be run for some checkpoints in the Thinkful curriculum. If you have trouble getting the server to run, reach out for assistance.
```

Comment:
Keeping this generic so they can reach out to whatever mode of assistance Thinkful currently offers (e.g., Slack channels)